bin/submit_build.py

```diff
@@ -72,6 +73,7 @@ def main():
     # Easybuild default paths
     # start using environment from local machine, job scripts get custom paths
     ebconf = {
+        'accept-eula-for': 'Intel-oneAPI,CUDA',
```

Comment:
this seems unrelated/unused here?

Reply:
indeed unrelated, i wanted to do this for a long time, so i sneaked it in.
we currently set this with an envvar in .bashrc, but better to let the script do it.